src/lsp/lp_lsp.ml

```diff
@@ -432,7 +463,7 @@ let main std log_file =
     let com = LIO.read_request stdin in
     LIO.log_object "read" com;
     process_input oc com;
-    F.pp_print_flush lp_fmt (); flush lp_oc;
```

Comment:
Thanks to keep `flush lp_oc` in a comment: `(*flush lp_oc*)`. Idem a few lines below.